app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -23,31 +29,52 @@
       to find other validators to delegate to.
     </div>
     <div v-if="delegation.loaded && undelegatedValidators.length > 0">
```

Comment:
do we need the distinction into `undelegatedValidators`?

Reply:
Not necessarily, Thanks! `yourValidators` will always be a superset of it, I'm gonna remove `undelegatedValidators`